apps/frontend/src/app/3_organisms/BorrowLoan/components/ExtendLoanForm/ExtendLoanForm.tsx

```diff
@@ -0,0 +1,394 @@
+                      )
+                    }
+                  />
+                }
+              />
+            </SimpleTable>
+
+            <div className="flex flex-row justify-between items-center mt-6 mb-3">
+              <div className="flex flex-row justify-start items-center gap-2">
+                <span>
+                  {t(translations.adjustCreditLine.labels.collateralRatio)}
+                </span>
+              </div>
+              <div className="text-primary-10">
+                <DynamicValue
+                  initialValue="0"
+                  value={collateralRatio.toString()}
+                  renderer={value => renderValue(value, '%', 0)}
+                />
+              </div>
+            </div>
+
+            <HealthBar
+              start={COLLATERAL_RATIO_THRESHOLDS.START}
+              middleStart={COLLATERAL_RATIO_THRESHOLDS.MIDDLE_START}
+              middleEnd={COLLATERAL_RATIO_THRESHOLDS.MIDDLE_END}
+              end={COLLATERAL_RATIO_THRESHOLDS.END}
+              value={collateralRatio.toNumber()}
+            />
+            {!isValidCollateralRatio && (
+              <ErrorBadge
+                level={ErrorLevel.Critical}
+                message={collateralRatioError}
+                dataAttribute="adjust-loan-collateral-ratio-error"
+              />
+            )}
+          </>
+        )}
+        {!useCollateral && (
+          <>
+            <div className="flex items-center mb-8 gap-3">
+              <AmountInput
+                value={depositAmount}
+                label={t(translations.common.amount)}
+                className="w-full flex-grow-0 flex-shrink"
+                placeholder="0"
+                readOnly
+              />
+              <div className="bg-gray-60 p-2.5 min-w-[5.5rem] rounded">
+                <AssetRenderer asset={debtToken} showAssetLogo />
+              </div>
+            </div>
+            <SimpleTable>
+              <SimpleTableRow
+                label={t(pageTranslations.labels.prepaidInterestAmount)}
+                value={
+                  <AmountRenderer
+                    value={depositAmount}
+                    suffix={getTokenDisplayName(debtToken)}
+                  />
+                }
+                valueClassName="text-primary-10"
+              />
+              <SimpleTableRow
+                label={t(pageTranslations.labels.newRolloverDate)}
+                value={renderNewRolloverDate}
+                valueClassName="text-primary-10"
+              />
+            </SimpleTable>
+          </>
+        )}
+
+        <div className="mt-6">
+          <SimpleTable>
+            <SimpleTableRow
+              label={t(pageTranslations.labels.liquidationPrice)}
+              value={
+                <DynamicValue
+                  initialValue="0"
+                  value={liquidationPrice.toString()}
+                  renderer={value => renderValue(value, collateralToken)}
```

Comment:
```suggestion
                  renderer={value => renderValue(value, debtToken)}
```
Liquidation price should be in `debtToken` (look at PRD).